CHANGELOG.rst

```diff
@@ -1,6 +1,13 @@
 ==========
 Change Log
 ==========
+`0.9.0`_ (2019-09-11)
+-----------------------
+* Add currency network registry that can be used to register curency networks and list registered networks
+* Change CurrencyNetworkInterface into an actual interface implementing ERC-165
+* Add function to track debt within currency networks, this can be used to represent a payment when no there is no path in between two addresses
+* Change function for executing meta-transaction in identity contracts to add tracking of debt (BREAKING)
```

Comment:
Change function for executing meta-transaction in identity contracts: Add fields to pay for transaction(BREAKING)